lib/active_model_serializers/model.rb

```diff
@@ -0,0 +1,43 @@
+# ActiveModelSerializers::Model is a convenient
+# serializable class to inherit from when making
+# serializable non-activerecord objects.
+module ActiveModelSerializers
+  class Model
+    include ActiveModel::Model
+    include ActiveModel::Serialization
```

Comment:
I ran this on my machine without error: https://gist.github.com/NullVoxPopuli/99da5afdc970c8036150

so, maybe I'm missing something. :-\